src/Homework_02/Dop_Prostoe_chislo.py

```diff
@@ -0,0 +1,12 @@
+n = int(input('Введите число\n'))
+if n == 1:
+    print('Число 1 не относится ни к простым, ни к составным числам')
+else:
+    i = 2
+# у составного числа есть делитель, не больше квадратного корня из числа
+    while i**0.5 <= n and n % i != 0:
+        i += 1
+    if i == n:
```

Comment:
if is our of while (bad indentation). it wouldn't work

Reply:
Fixed